src/ssl.c

```diff
@@ -4053,6 +4053,29 @@ int wolfSSL_GetHmacSize(WOLFSSL* ssl)
     return BAD_FUNC_ARG;
 }
 
+int wolfSSL_GetPeerSequenceNumber(WOLFSSL* ssl, word64 *seq)
+{
+    if (ssl) {
+        *seq = (((word64)ssl->keys.peer_sequence_number_hi << 32) |
+                ssl->keys.peer_sequence_number_lo);
+        return 0;
```

Comment:
Please rewrite with a single `return` at bottom. Thanks! Same for `wolfSSL_GetSequenceNumber`.

Reply:
I guess you mean to keep the 1st ssl check for the assignment and do a `return ssl ? 0 : BAD_FUNC_ARG`. To avoid the double check what do you think about the following 1 statement way?

```
int wolfSSL_GetPeerSequenceNumber(WOLFSSL* ssl, word64 *seq)
{
    return ssl ? !(*seq = (word64)ssl->keys.peer_sequence_number_hi << 32 |
                ssl->keys.peer_sequence_number_lo) : BAD_FUNC_ARG;
}
```